src/LaunchDarkly.EventSource/LaunchDarkly.EventSource.csproj

```diff
@@ -1,23 +1,24 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-
   <PropertyGroup>
     <Version>4.0.0-alpha.5</Version>
     <TargetFrameworks>netstandard2.0;net452</TargetFrameworks>
-    <LicenseExpression>Apache-2.0</LicenseExpression>
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
     <AssemblyName>LaunchDarkly.EventSource</AssemblyName>
     <DebugType>portable</DebugType>
     <OutputType>Library</OutputType>
+    <LangVersion>7.3</LangVersion>
     <PackageId>LaunchDarkly.EventSource</PackageId>
     <Company>LaunchDarkly</Company>
+    <Authors>LaunchDarkly</Authors>
+    <Owners>LaunchDarkly</Owners>
     <Copyright>Copyright 2017 Catamorphic, Co.</Copyright>
-    <Description>LaunchDarkly .NET Event Source library for Server Sent Events</Description>
+    <Description>LaunchDarkly Server-Sent Events client</Description>
     <PackageProjectUrl>https://github.com/launchdarkly/dotnet-eventsource</PackageProjectUrl>
     <RepositoryUrl>https://github.com/launchdarkly/dotnet-eventsource</RepositoryUrl>
-    <SignAssembly>true</SignAssembly>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="LaunchDarkly.Logging" Version="1.0.0" />
+    <PackageReference Include="LaunchDarkly.Logging" Version="[1.0.0,2.0.0)" />
```

Comment:
I've been moving toward using dependency version ranges instead of single versions, for dependencies like this that are referencing an API whose actual version is expected to be overridden by the application or by a further-up library (as opposed to something like the dependency from `LaunchDarkly.ServerSdk` to `LaunchDarkly.InternalSdk`, where the latter is conceptually an integral part of the former and an exact version dependency is appropriate). I'm not sure it makes a big difference in practice, as the default behavior (in .NET Core at least) is already to allow lower versions of a transitive dependency to be overridden by higher ones, but I think this makes the intention clearer.